feature: Add default path of /etc/rc.conf when not using any arg

gyptazy opened the pull request:
feature: Add default path of /etc/rc.conf when not using any arg

 * Use default path of `/etc/rc.conf` when not using any arg
 * Validate that default path exists in case of execution on any other system

Fixes: #9

scovl commented:
Hey @gyptazy, thanks for the PR! It looks great for handling the `/etc/rc.conf` default and checking if the file exists on other systems. There’s just one small snag: the CI fails because there’s **no newline at the end of `main.c`**. With `-Werror`, that tiny warning becomes a full-blown error. Just add a quick newline (press Enter at the end of the file), and it should be all good!

A couple of other small tips:

1. Watch out for pointer arithmetic:  
   ```c
   // Wrong (this is pointer addition)
   file_path + default_file;

   // Right (actual assignment)
   file_path = default_file;
   ```
2. If you don’t specifically need `access(file_path, R_OK)`, you can just rely on `fopen()` failing if the file doesn’t exist or isn’t readable.  

Anyway, thanks again for your work—looking forward to merging this once the newline fix is in!

gyptazy commented:
Hey @scovl,

thanks, just added and squashed.

Thank you very much for the other tips, really appreciate! Feel free to edit this PR to fit your needs and acceptance criteria!

Thanks,
gyptazy
